### DISCUSSION TITLE
How to evaluate a curve?

### DISCUSSION THREAD

**Discussion by Phenzmnn** [CadQuery/cadquery#1302] [Q&A] [answered]
Hello all!
I'm new to CadQuery because I am looking for a FOSS alternative to Rhinoceros3D/Grasshopper.
One major feature I am looking for is "evaluating" a curve. 

What I want is to get a point on a curve along with its tangent vector by inputting a fraction from 0-1.

I would like use the output to position and align a new workplane. 

Is this already possible with CQ? 

Thanks for your help!

**Comment by adam-urbanczyk** [comment, marked as answer]
`cq.Edge.positionAt` and `cq.Edge.tangentAt`. Also works for `cq.Wire`
